sections/04_study.md

```diff
@@ -30,78 +30,67 @@ and why it has received less attention, @tag:Chen2015_trans_species as example`
+linear regression in inferring the expression of over 20,000 target genes based
+on a representative, well-connected set of about 1,000 landmark genes
+[@tag:Chen2016_gene_expr]. However, while the deep learning model outperformed
+existing algorithms in nearly every scenario, the model still displayed poor
+performance. The paper was also limited by computational bottlenecks that
+required data to be split randomly into two distinct models and trained
+separately. It is unclear how much performance would have increased if not for
+computational restrictions. Alternatively, epigenetic data may have sufficient
+explanatory power for inference of gene expression: for instance, a
 convolutional neural network applied to histone modifications, termed
-DeepChrome, [@tag:Singh2016_deepchrome] was shown to predict gene expression
-output. DeepChrome greatly improved high or low expression prediction accuracy
-over existing methods. Deep learning applied to epigenetic data for gene
-expression inference is a promising approach to study gene regulation. Deep
-learning approaches have also been applied to study cancer gene expression data
-with goals of identifying subtypes of patients with different molecular features
-and clinical manifestations [@tag:Liang2015_exprs_cancer]. In the study, the
-authors combine RBMs to integrate gene expression, DNA methylation, and miRNA
-data and use the constructed features in search of ovarian cancer subtypes.
-While the aforementioned approaches are promising, many convert gene expression
-measurements to categorical or binary variables thus ablating many complex gene
-expression signatures present in intermediate and relative numbers.
-
-Deep learning applied to gene expression data is in its infancy but the future
-is bright. Many hypotheses can now be interrogated because of increasing
-amounts of data and new developing technologies. For example, there is a
-growing appreciation for the large impact of disease heterogeneity on research
-and treatment strategies for diseases. New technologies are being developed,
-such as single cell RNA-seq and high throughput fluorescence based imaging that
-are good matches for deep learning. Concurrently, deep learning methods are
-being developed to address novel problems such as adjusting for batch effects in
-single-cell RNA-seq data [@tag:Shaham2016_batch_effects]. Moreover, deep
-learning is already well established in the image processing community, so the
-marriage of fluorescence based imaging techniques and deep learning is natural.
-These technologies are growing in popularity and will provide increasingly novel
-perspectives with respect to how cellular heterogeneity impacts gene expression
-coordination within a sample. In general, as the flow of gene expression data
+DeepChrome, [@tag:Singh2016_deepchrome] was shown to improve prediction accuracy
+over existing methods.  Deep learning can also be useful for integrating
```

Comment:
Was the phrase "high or low" important?  Did this indicate that they predicted binary expression?

If so, we could reincorporate it by saying "prediction accuracy of binary expression over..." or "of high and low expression over..." to indicate that this is a simpler setting that continuous expression.